nautobot/docs/additional-features/template-filters.md

```diff
@@ -155,6 +155,23 @@ Render a muted placeholder (`<span class="text-muted">&mdash;</span>`) if value
 {{ html | placeholder }}
 ```
 
+### render_boolean
+
+Render HTML from a computed boolean value. 
+
+If value is truthey - a string value of either 'y', 'yes', 't', 'true', 'on' or '1' (or their equivalent uppercase
+values) -- or True / > 0 -- this
+renders `<span class="text-success"><i class="mdi mdi-check-bold" title="Yes"></i></span>`
+
+If value is falsey - a whitespace-only string, an empty string or string with a value of either
+'n', 'no', 'f', 'false', 'off', or '0' (or their equivalent uppercase values) -- or False / 0 -- this renders `<span class="text-danger"><i class="mdi mdi-close-thick" title="No"></i></span>`
```

Comment:
Thinking about this again... are there actually cases in the existing templating where we need the `is_truthy` functionality, or was this a new enhancement you added to the filter? If the latter, I'd be tempted to suggest that we simplify things and remove that logic from the filter, just letting empty strings evaluate as False and all other strings evaluate as True. Happy to discuss this question further.

Reply:
You're right.. I just noticed the is_truthy functionality in the codebase and thought it might be good to use that as an enhancement for this feature, but I don't think there's a need for it.